spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -991,7 +992,7 @@ def after_hooks(source)
 
           # rubocop:disable AlignArray
           expect( formatter.legacy_messages ).to eq [
-              :before_features,
+            :before_features,
```

Comment:
Perhaps someone already noted this, but all the places where the rule AlignArray has been explicitly disabled, those places in the code are carefully arranged ASCII art of a sort. 

The code indentation there explains levels of nesting and ordering. 

Could you make those align in a harmonious way in this change? I mean: Avoid four-space indents.

Reply:
I certainly can. Are you concerned with this even if all tests are passing? There are other instances of this are all on the same indent and they work as well. 

Matt's input on this is that the tabs are more of a guideline for visibility but aren't actually required. 

Let me know your thoughts. :)

Reply:
I care enough about the previous effort of “sketching the structure” using indents that I would like for it to be kept. 

This is a nonfunctional design detail but a quite good one. It communicated things to me, at least.

Reply:
I would like to point out that the legacy_api module will be deleted as soon as possible (we just have to rewrite the pretty and html formatter first, see https://github.com/cucumber/cucumber-ruby/issues/839), so any work in the legacy_api or its spec it a waste of time. I would recommend to remove all changes in the legacy_api module and its spec from this PR (and any other similar PR).

Reply:
Should I do a wholesale exclude on the legacy api file?

Reply:
@olleolleolle fair enough! I'm afk for a bit but I'll get to this when I can. Thanks for the input guys! (You too @brasmusson !)